### ISSUE TITLE
Reduce Master Node Token Lockup to 7 or 10 days

### ISSUE THREAD

**Issue by presman** [Beldex-Coin/beldex#56]
There is large support in the Master Node community to reduce the token lockup period from 30 days to a reasonable period of 7 or 10 days.  While the system continues to be enhanced, a deregistered node owner must wait for 30 days to re-register the node and continue contributing to the blockchain stability and transaction validations.  

A lockup period of 7 or 10 days still provides commitment to the project and validations of blocks.    We should show support to the community by helping them to get deregistered nodes up as quickly as possibly and not lead them to frustrations that abandon the project.

We respectfully ask that you consider this development change as as possible.

presman

**Comment by Nectum8** [comment]
Please listen to your community. This rule could destroy the project, even if you added It to bring estabilty to your projects, could do the oppsite effect.

**Comment by lucablighthl** [comment]
Yes, that change would be something really good for the masternode owners.

**Comment by rage0674** [comment]
I support this request.  If I have to wait 30 days it’s an unreasonably long time for an issue that is not my fault. Please consider a solution of reduced lock up.

**Comment by codeman-crypto** [comment]
We will consider this request and come up with a solution for the same.

**Comment by joshuaolufs** [comment]
What if instead of a boolean locked/unlocked state there was a third option for locked but unassigned coins. So you would have 3 states.

1) Locked and Assigned - Coins are completely locked. Become 'Locked and Unassigned' if masternode deregisters.

2) Locked and Unassigned - Coins can only be reassigned to a masternode, with no waiting period. Become Unlocked in 7-10 days if not reassigned.

3) Unlocked - Coins can be used without restriction.

This would allow masternode owners to keep their nodes up without any unnecessary downtime while still ensuring system stability.

**Comment by rpraghas-create** [comment]
I support this request.

**Comment by lucablighthl** [comment]
> What if instead of a boolean locked/unlocked state there was a third option for locked but unassigned coins. So you would have 3 states.
> 
> 1. Locked and Assigned - Coins are completely locked. Become 'Locked and Unassigned' if masternode deregisters.
> 2. Locked and Unassigned - Coins can only be reassigned to a masternode, with no waiting period. Become Unlocked in 7-10 days if not reassigned.
> 3. Unlocked - Coins can be used without restriction.
> 
> This would allow masternode owners to keep their nodes up without any unnecessary downtime while still ensuring system stability.

Very nice suggestions! With something like this any kind of problem that happens to a masternode could be easily solved.

**Comment by PanderMusubi** [comment]
I had several masternodes that got deregistered because of the systemd watchdog that restarted services too often. Unlocking after 7 days would be more fair as most of these situations are related to that or change of IP address, not to malicious behavior. Decreasing the lock time would also promote more in the total of masternodes being up (more quickly).
